modules/mri_upload/js/mri_upload.js

```diff
@@ -115,34 +114,38 @@ function uploadFile() {
                 function (evt) {
                     if (evt.lengthComputable) {
                         var progressbar = $("#progressbar"),
-                            progresslabel = $("#progresslabel"),
-                            percent = Math.round((evt.loaded / evt.total) * 100);
-                        progressbar.attr('value', percent);
-                        progresslabel.text(percent + "%");
+                        progresslabel = $("#progresslabel"),
+                        percent = Math.round((evt.loaded / evt.total) * 100);
+                        $(progressbar).width(percent + "%");
+                        $(progresslabel).html(percent + "%");
+                        progressbar.attr('aria-valuenow', percent);
                     }
                 },
                 false
             );
             return xhr;
+        },
+        success: function (data) {
+            alert("File Uploaded");
```

Comment:
This could be replaced by some kind of CSS element (something cleaner). It can be discussed in more details.